Misc/NEWS.d/next/Library/2020-12-03-00-40-12.bpo-42547.O2BJhp.rst

```diff
@@ -0,0 +1 @@
+argparse: fixed `add_argument(metavar=<tuple>, nargs='+')` for positional arguments
```

Comment:
Please add more details and use full sentences. Code snippets should be enclosed in double backquotes.

Please replace "argparse" with
```rst
:mod:`argparse`
```